### PR TITLE
INSTALL-git.md: Update homebrew section

### PR DESCRIPTION
**Pull request by davelab6**
https://github.com/fontforge/fontforge/issues/2296

### CODE REVIEW

**Comment by larsenwork** [comment]
Sounds a little confusing telling people they can't use Homebrew and then moves on to use brew commands - could maybe be clarified a bit:)

**Comment by davelab6** [comment]
They can use homebrew for the python, but not the ui

File: INSTALL-git.md [hunk 1]
```diff
@@ -154,3 +126,41 @@ Then edit then your shell profile (eg, `~/.bash_profile` or `~/.zprofile`) and a
 ```
 export PATH=${PATH}:/usr/local/opt/gettext/bin
 ```
+
+## On Mac OS X with Homebrew
+
+Homebrew no longer packages FontForge with a UI, but the release GUI app bundles are available from Cask. 
+You can install FontForge from the current github source code using a package manager, such as MacPorts or Fink. 
+
+Here is how to install FontForge without a UI:
```
Comment (larsenwork) [inline INSTALL-git.md]: Maybe make this

##Install FontForge without UI using HomeBrew

Comment (davelab6) [inline INSTALL-git.md]: Make the PR :)

**Pull request merged**